.github/workflows/sync-api.yml

```diff
@@ -0,0 +1,25 @@
+name: Syncing API docs
+on:
```

Comment:
Could add a blank line after `name` and `on`? (So that we separate the sections. Check the other workflows as examples).

Also could you add a blank line at the end of the file? (EOL)

Reply:
done!